src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -45,6 +45,29 @@ namespace dpdk {
     return std::bitset<CHAR_BIT * sizeof(n)>{n}.count();
   }
 
+  uint16_t eal::get_first_slave_coreid(const char *coremask)
+  {
+    uint64_t ulCoreMask;
+    uint16_t usBitCount;
+    uint16_t first_slave_coreid;
+    int iRet = 0;
+
+    iRet = sscanf(coremask, "%lx", &ulCoreMask);
+    if (1 != iRet || 0 == ulCoreMask)
```

Comment:
nit, i'd suggest avoid using yoda conditions. as modern compiler will print warnings like
```
warning: suggest parentheses around assignment used as truth value [-Wparentheses]
```
 at seeing 
```c++
if (foo = 0) {
  heal(the_world);
}
```

Reply:
now it's this:
auto core_value = stoull(coremask_arg, nullptr, 16);
and ,thers is no warning.